lib/signaling/track.js

```diff
@@ -122,11 +130,34 @@ class TrackSignaling extends EventEmitter {
     }
     return this;
   }
+
+  /**
+   * Updates track switch on/off state.
+   * @param {boolean} isSwitchedOff
+   * @returns {this}
+   */
+  switched(isSwitchedOff) {
```

Comment:
@makarandp0 This should move to `lib/signaling/remotetrackpublication.js`. Also nitpick: `setSwitchedOff()`?